web_01.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="./css/style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+        <title>Website_02: Mini</title>
+    </head>
+
+    <body>
+        <header>
+            <div class="main__header" id="main_header">
```

Comment:
You can use `<header>` as selector in styling instead of creating excess `div.main__header`